konfera/admin.py

```diff
@@ -129,8 +129,23 @@ class OrderAdmin(admin.ModelAdmin):
 admin.site.register(Order, OrderAdmin)
 
 
+class TicketTypeAdmin(admin.ModelAdmin):
+    list_display = ('title', 'price', 'attendee_type', 'event', 'status')
+    list_filter = ('attendee_type',)
+    readonly_fields = ('status',)
+    fieldsets = (
+        (_('Details'), {
+            'fields': ('title', 'description', 'price', 'attendee_type', 'event',)
+        }),
+        (_('Availability'), {
+            'fields': ('date_from', 'date_to', 'status')
```

Comment:
instead of status you can use get_status or something similar defined in comment in model.